Feature home feed

NikolasGeorgaklis opened the pull request:
User data is now immediately updated in parse when the user opens the app. I added a function in HomeViewController updates user rank, correct picks, ratio of correct picks to total, and the accuracy percentage of each user. This data is subsequently shown in the home feed and/or the user's profile.

NikolasGeorgaklis committed:
Update user stats onload in home feed

Function in HomeViewController updates user rank, correct picks, ratio of correct picks to total, and the accuracy percentage of each user. This data is subsequently shown in the home feed and/or the user's profile.

NikolasGeorgaklis committed:
Update Breakpoints_v2.xcbkptlist

NikolasGeorgaklis committed:
Delete Breakpoints_v2.xcbkptlist

vaguilar commented on NikoBracket/HomeViewController.m:
```diff
@@ -12,6 +12,15 @@
 #import "HomeViewCell.h"
 #import "UIImageView+AFNetworking.h"
 
+static NSMutableArray *teamsInTournament;
```
any reason to make this static const? i think this makes more sense as a property on HomeViewController

vaguilar commented on NikoBracket/HomeViewController.m:
```diff
@@ -12,6 +12,15 @@
 
+static NSMutableArray *teamsInTournament;
+static int numOfPastGames;
+static int kRound1 = 0;
+static int kRound2 = 1;
+static int kRound3 = 2;
+static int kRound4 = 3;
+static int kSemiFinal = 4;
+static int kFinal = 5;
```
are these the same as from the create bracket controller? maybe it does make sense to have a constants file for these.

vaguilar commented on NikoBracket/HomeViewController.m:
```diff
@@ -71,4 +81,143 @@ - (NSInteger)tableView:(nonnull UITableView *)tableView numberOfRowsInSection:(N
+    NSDictionary *dict = [NSDictionary dictionaryWithContentsOfFile: pListPath];
+    NSString *key = [dict objectForKey: @"API_Key"];
+    NSError *error;
+    NSString *url_string = [NSString stringWithFormat: @"%@%@",  @"https://api.sportsdata.io/v3/cbb/scores/json/Tournament/2022?key=", key];
```
maybe this can also be in the consts file

NikolasGeorgaklis committed:
Add search bar to home feed so user can search for accounts

NikolasGeorgaklis committed:
Display final four teams along with regions

Now when a user taps on a user in the home feed, a view controller is displayed modally which shows the user's final four picks from each region.

NikolasGeorgaklis committed:
Add user info to home feed in detail view

A user can now view another user's profile, info, picks, and stats from the home feed.

NikolasGeorgaklis committed:
Merge branch 'main' into feature_homeFeed
